### ISSUE TITLE
Asymmetric thickening of the nipple-areolar complex

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10086]
**Preferred term label:**
Asymmetric thickening of the nipple-areolar complex

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0032314

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary Paget disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000058
- label: Asymmetric thickening of the nipple-areolar complex
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: PMID:26385836
- def: Thickening of the skin around the nipple.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10086
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO ID corrected to HP:0040405
